server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -923,4 +923,10 @@ timestamp_precision=ms
 ### Group By Fill Configuration
 ####################
 # Datatype: float
-# group_by_fill_cache_size_in_mb=1.0
\ No newline at end of file
+# group_by_fill_cache_size_in_mb=1.0
+
+####################
+### password encrypt Configuration
+####################
```

Comment:
Hi Chao, any user guide about how to use the configuration?

Reply:
By default, MD5 encryption is used. We can extend our own encryption algorithm by implementing the AsymmetricEncrypt interface and adding this class to the configuration item. If a parameter is required, add the parameter to the iotdb_server_encrypt_decrypt_provider_parameter configuration item because some encryption algorithms require the parameter. Later, I will contribute a SHA384 encryption algorithm. This algorithm has a high encryption level and meets the high algorithm requirements. This algorithm requires parameters.
I will add some user guide in doc.
finally, happy new year for you.